docs/cloud-native-security/vuln-management-dashboard.asciidoc

```diff
@@ -0,0 +1,26 @@
+[[vuln-management-dashboard]]
+= Cloud Native Vulnerability Vulnerability Management Dashboard
+
+The Cloud Native Vulnerability Management (CNVM) dashboard gives you an overview of the vulnerabilities detected in your cloud infrastructure by the CNVM integration.
```

Comment:
Correct me if I'm wrong, but isn't installing the CNVM integration a pre-req for using this dashboard? If it is, I recommend moving that information to the requirements block [below](https://github.com/elastic/security-docs/pull/3553/files#diff-77efc2779f8dae31fd438b1a7e66da6edff45c816b825d4586825d66c5c26aabR10) and delete the note on line 6. 
 
```suggestion
The Cloud Native Vulnerability Management (CNVM) dashboard gives you an overview of your cloud infrastructure vulnerabilities.
```